lib/command/index.js

```diff
@@ -189,7 +189,7 @@ class Command extends Registry {
    * @type string
    **/
   get usage() {
-    return usage.from(this)
+    return usage.from(this, this.parsed?.color || true)
```

Comment:
I'm still not sure color is working for me.  I noticed this a while ago, but even adding it here didn't seem to do anything different on my screen.

Reply:
This should be `!this.argv.color`

parsed is the cache, `argv` is the getter that does the parsing and populates `parsed` so `argv` always return the flag input

Reply:
Strange, when it becomes `!this.argv.color`, it throws when trying to use `--help`

```bash
> node ./pipetool.js bulk-config-gen --help
UnknownFlagException: unknow field: help.
    at get argv [as argv] (/Users/darin.spivey/github/node-seeli/lib/command/index.js:234:23)
    at get usage [as usage] (/Users/darin.spivey/github/node-seeli/lib/command/index.js:192:43)
```